### ISSUE TITLE
Request improved guidance to using *cache* files on Android

### ISSUE THREAD

**Issue by SomeTroglodyte** [libgdx/libgdx#6529]
#### Issue details
The [Documentation](https://github.com/libgdx/libgdx/wiki/File-handling#file-storage-types) does not clearly answer the question "I want to store files that will disappear when the user taps the 'Clear cache' button in App Info - Storage. How do I apply the Gdx Api to achieve this?" - the answer might be "prepend '../cache/' to your desired path and use Gdx.files.local", which is not really complicated. But this is platform-specific internal knowledge.

So, some more documentation might improve libGdx, an abstraction in the api might also help some developers.

#### Version of LibGDX and/or relevant dependencies
1.10.0

#### Workaround (kotlin)
```kotlin
/**
 *      This extension function returns FileHandles into the application-private cache area,
 *      such that files created there will _disappear_ using Android's App-Info (settings) Storage->Clear Cache button.
 *
 * @param   path: a relative file path to a file or directory which may exist or not
 * @return  FileHandle accessor to the specified filesystem location
 */
private fun com.badlogic.gdx.Files.cache(path: String): FileHandle {
    return if(Gdx.app.getType() == ApplicationType.Android)
            Gdx.files.local("../cache").child(path)
        else Gdx.files.local(path)
}
```

#### Please select the affected platforms
- [x] Android
- [ ] iOS
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [ ] MacOS

**Comment by SimonIT** [comment]
This PR #6246 adds support for cache files

**Comment by MrStahlfelge** [comment]
Yep, but there was no interest in the addition expressed so the PR is stalled.
